app/ForkedStar.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import { env } from "@/env.mjs";
+import { useEffect, useMemo, useState } from "react";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+
+function ForkedStar(props: any) {
+  const [data, setData] = useState<any>();
+  let forkCount = 0;
+  let starCount = 0;
+
+  useMemo(async () => {
+    const res = await fetch(
+      `https://api.github.com/orgs/${env.NEXT_PUBLIC_GITHUB_ORG}/repos`,
+    );
+    setData(await res.json());
+  }, []);
+
+  data?.map((ls: any) => {
+    if (ls.forks) {
+      forkCount += ls.forks;
+    }
+    if (ls.stargazers_count) {
+      starCount += ls.stargazers_count;
+    }
+    // console.log(ls.full_name, ls.forks, ls.stargazers_count);
+  });
+  console.log(forkCount);
```

Comment:
Let's make this as Server component instead of a client component

Reply:
Okay so it means I can use getServerSideProps to make it server sider rendering?

Reply:
Those are relevant to pages router only. These are app router pages.

Check out NextJS docs on App Router
https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-server-with-fetch